src/test/ui/did_you_mean/bad-assoc-ty.stderr

```diff
@@ -130,7 +143,97 @@ LL | fn foo<X: K<_, _>>(x: X) {}
    |             |
    |             not allowed in type signatures
 
-error: aborting due to 20 previous errors
+error[E0121]: the type placeholder `_` is not allowed within types on item signatures
+  --> $DIR/bad-assoc-ty.rs:49:13
+   |
+LL | fn foo<X: K<_, _>>(x: X) {}
+   |             ^  ^ not allowed in type signatures
+   |             |
+   |             not allowed in type signatures
+   |
+help: use type parameters instead
+   |
+LL | fn foo<X, T: K<T, T>>(x: X) {}
+   |         ^^^    ^  ^
```

Comment:
This is now duplicated, sadly.

Reply:
As a result of addressing review comments?